ci/.gitlab-ci.yml

```diff
@@ -15,5 +15,6 @@ include:
   - local: 'ci/cuda/gcc11_release_scalapack.yml'
   - local: 'ci/cuda/gcc11_codecov.yml'
   - local: 'ci/cuda/gcc11_debug_scalapack.yml'
+  - local: 'ci/cuda/gcc12_release_stdexec.yml'
```

Comment:
Could you rename the file and pipeline names to use `gcc13` now that the compiler version is different please?